holidays/countries/hungary.py

```
@@ -131,13 +150,83 @@ class HungaryStaticHolidays:
+        1997: (
+            (MAY, 2, APR, 26),
+            (OCT, 24, OCT 18),
+            (DEC, 24, DEC, 20),
+        ),
+        1998: (
+            (JAN, 2, JAN, 10),
+            (AUG, 21, AUG 15),
+            (DEC, 24, DEC, 19),
+        ),
+        1999: (
+            (DEC, 24, DEC, 18),  
```

Ruff (0.11.9)

Trailing whitespace

(W291)